src/structures/User.js

```diff
@@ -98,6 +98,16 @@ class User {
   }
 
   /**
+   * Get the note set for the user
+   * <warn>This property is meant for user accounts only!</warn>
```

Comment:
Should be changed to be consistent with the other warnings (`This is only available for user accounts.`)